PayItForward.ConsoleClient.UnitTests/Web/Controllers/HomeController/Index_Should.cs

```diff
@@ -41,7 +44,7 @@ public void ReturnIndexViewModel_WithAListOfBasicStoryViewModel()
                 .Returns(HomeController_Stubs.GetTestListOfStoryDtos());
 
```

Comment:
You can check the count of returning stories, because if the Id is negative, it won't contain any basic stories

Reply:
Nice suggestion!